README.md

```diff
@@ -92,27 +99,10 @@ conda install -c conda-forge pre_commit
 pre-commit install
 
 ```
-
-The above environment allows GRACE to be run on previously annotated images (saved on .grace format). If, however, annotation is needed, use the following lines to load and activate the correct environment, which will install further packages (such as napari):
-
-```sh
-# create the playground from environment_annotation.yml
-conda env create -f environment_annotation.yaml
-
-# To activate this environment, use
-#
-#     $ conda activate grace-annotation
-#
-# To deactivate an active environment, use
-#
-#     $ conda deactivate
-
-conda activate grace-annotation
-
-```
-
 ---
 
+If you currently do not have any data to test / implement GRACE on, have a look at the option of **simulating a synthetic dataset** as described in [this](./grace/simulator/README.md) README. An assessible link to some pre-annotated simulated images is coming soon! 🚧
```

Comment:
accessible?

Reply:
The one I created in on the Turing OneDrive, we need to agree how to share the data (usually not the best practice to have data in the repo, as they are quite large files).